docs/use/db_provision.md

```diff
@@ -2,8 +2,13 @@
 
 Provisioning a database instance involves setting up and configuring a database instance on the Percona Everest platform.
 
+
+!!! warning
+    Do not modify the root user's password in the database, as this could potentially break your cluster
```

Comment:
Don't forget to update this one accordingly to what we just added to the known limitations

Reply:
@recharte I've added pls check

Reply:
Are you sure? I don't see it...